src/dapp-tests/pass/abstract.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.6.7;
+
+import {DSTest} from "ds-test/test.sol";
+
+// should not be run (no code)
+abstract contract MyTest is DSTest {}
```

Comment:
```suggestion
abstract contract MyTest is DSTest {
    function abstractTest() public {
        assertTrue(true);
    }
}
```
Could we add an abstract test? This is exactly what I needed abstract tests for.